src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/NuGetProjectType.cs

```diff
@@ -28,10 +28,7 @@ public enum NuGetProjectType
         /// </summary>
         UwpProjectJson = 3,
 
-        /// <summary>
-        /// Corresponds to <see cref="ProjectKNuGetProjectBase"/>.
-        /// </summary>
-        XProjProjectJson = 4,
+        // TODO NK - More cleanup?
```

Comment:
remove todo, or replace with a link to an issue

Reply:
removed.